config/initializers/merritt.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+Rails.application.config.merritt_user = "tjohnson"
```

Comment:
We need to scrub these commits immediately. These credentials are private!

Reply:
To document this here: I changed the credentials on the Merritt-side immediately following this comment.

Reply:
Config under `config/initializers/merritt.rb` has now moved to `config/merritt.yml`